src/rednoise.c

```diff
@@ -3,12 +3,20 @@
 #include "rednoise_cmd.h"
 #ifdef USEDMALLOC
 #include "dmalloc.h"
+#include "misc_utils.h"
```

Comment:
Note that this include is inside the #ifdef corresponding to the dmalloc.h file!  Should not be there.